ddtrace/_trace/tracer.py

```diff
@@ -416,17 +417,17 @@ def get_log_correlation_context(self) -> Dict[str, str]:
         if self.enabled or self._apm_opt_out:
             active = self.context_provider.active()
 
+        trace_id = 0
+        span_id = 0
         if isinstance(active, Span) and active.service:
             service = active.service
         else:
             service = config.service
         if active:
-            trace_id = active.trace_id
-            if config._128_bit_trace_id_enabled and not config._128_bit_trace_id_logging_enabled:
-                trace_id = active._trace_id_64bits
+            trace_id, span_id = _get_trace_details_for_log_injection(active)
         return {
-            "trace_id": str(trace_id) if active else "0",
-            "span_id": str(active.span_id) if active else "0",
+            "trace_id": str(trace_id),
+            "span_id": str(span_id),
```

Comment:
With the refactor below we do not need to convert these values to strings:

```suggestion
            "trace_id": trace_id,
            "span_id": span_id,
```